course6/toyapp_mikhail/displaylib/cube.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include "matrix.h"
+#include "vector2d.h"
+#include "vector3d.h"
+#include "surface3d.h"
+#include "rgb.h"
+
+namespace Display {
+
+	class Cube
```

Comment:
Indenting the entire namespace creates huge wasted area that spans the entire file, and doesn't improve readability.